map/track.cpp

```diff
@@ -220,6 +220,12 @@ dp::Color Track::GetColor(size_t layerIndex) const
   return dp::Color(m_data.m_layers[layerIndex].m_color.m_rgba);
 }
 
+void Track::SetColor(dp::Color color)
+{
+  m_isDirty = true;
```

Comment:
Does it make sense to check for color equality here?

Reply:
Probably yes, but I am following the Bookmark editing flow:

```
void Bookmark::SetColor(kml::PredefinedColor color)
{
  SetDirty();
  m_data.m_color.m_predefinedColor = color;
}
```